consensus/enclave/measurement/build.rs

```diff
@@ -70,8 +70,10 @@ fn main() {
     }
 
     builder
-        .cargo_builder
-        .target_dir(env.target_dir().join(CONSENSUS_ENCLAVE_NAME));
+        .target_dir(env.target_dir().join(CONSENSUS_ENCLAVE_NAME).as_path())
+        .add_rust_flags(&["-D", "warnings"])
```

Comment:
This can't come from a `.cargo/config` file? Its occasionally nice to be able to disable this when developing.